docs/language-requests.rst

```diff
@@ -19,7 +19,7 @@ time to time in the Highlight.js issue tracker and discussion group.
     This means that *there's no point in requesting a new language without also
     providing a 3rd party implementation* (we'll simply close "Please support
     language Xyz" issues with a link to this explanation). If you'd like to see
-    a particular language available but cannot implement it, the best way to
+    a particular language available that you cannot implement yourself, the best way to
```

Comment:
perhaps using "unable" here could be better?

Reply:
Why?